tests/test_tools/test_convert_dgv.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pytest
+from unittest.mock import Mock, patch
+from tools.convert_dgv import main as convert_dgv_main
+import sys
+
+
+@pytest.mark.parametrize(
+    "filename,expected_file",
+    [
+        ["dgv_test.tab", "dgv_test_expected.tab"],
+    ],
+)
+def test_dgv_examples(tmp_path, filename, expected_file):
+    data_dir = os.path.join(os.path.dirname(__file__), "data")
+
+    output_path = str(tmp_path / "tmp_data.tab")
+    args = [
+        "python",
+        "--input",
+        os.path.join(data_dir, filename),
+        "--output",
+        output_path,
+    ]
+
+    with patch.object(convert_dgv_main, "main", create=True):
+
+        with patch.object(sys, "argv", args) as m:
+            convert_dgv_main()
+
+    with open(os.path.join(data_dir, expected_file), 'r') as fh:
+        expected = fh.read().replace('\n', '')
```

Comment:
i don't think you need to replace the new lines do you?

Reply:
I thought it wouldn't either, but there seems to be something wrong if new lines aren't replaced `\n` are not replaced.  But the content should be the same. 
```
E       AssertionError: assert 'tracking_id\...tFalse\tbed\t' == 'tracking_id\...alse\tbed\t\n'
E         Skipping 6205 identical leading characters in diff, use -v to show
E         - 	False	bed	
E         ? 	     	   	-
E         + 	False	bed
```

Reply:
maybe it is just the last '\n' character? have you tried strip() instead of replacing all the new lines? or adding the newline to the test file

Reply:
Addressed in [8f1cffb](https://github.com/bcgsc/mavis/pull/336/commits/8f1cffb6e373f828c0cf0c74b796d73bdd3bcd5c#diff-fcc5f943cc3cb9d2395934b743dd5cfdf04be0ef2db3bfcc46331086b7d4bed5R32).